### ISSUE TITLE
Critical Start-up Failure: /usr/local/lib/python2.7/dist-packages/M2Crypto-0.25.1-py2.7-linux-x86_64.egg/M2Crypto/__m2crypto.so: undefined symbol: SSLv2_method

### ISSUE THREAD

**Issue by raveydaveygravy** [smicallef/spiderfoot#127]
I am seeing the error below when I execute sf.py. As you can see from the **pip list** output, I have M2Crypto installed. Any ideas on how I can fix this please?

```
user01@recon02:~/spiderfoot$ ./sf.py

Critical Start-up Failure: /usr/local/lib/python2.7/dist-packages/M2Crypto-0.25.1-py2.7-linux-x86_64.egg/M2Crypto/__m2crypto.so: undefined symbol: SSLv2_method
=================================
It appears you are missing a module required for SpiderFoot
to function. Please refer to the documentation for the list
of dependencies and install them.

Python modules required are:
 - M2Crypto
 - netaddr
 - dns
 - cherrypy
 - mako
 - socks
 - pyPdf
 - metapdf
 - openxmllib
 - stem
 - bs4
 - gexf
 - phonenumbers

If you are running on Windows and getting this error, please
report this as a bug to support@spiderfoot.net.

user01@recon02:~/spiderfoot$ pip list
DEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
adium-theme-ubuntu (0.3.4)
cffi (1.5.2)
chardet (2.3.0)
cheroot (5.1.0)
CherryPy (10.1.1)
cryptography (1.2.3)
dicttoxml (1.6.6)
dnspython (1.12.0)
enum34 (1.1.2)
GeoIP (1.3.2)
idna (2.0)
ipaddress (1.0.16)
jsonrpclib (0.1.3)
lxml (3.7.2)
M2Crypto (0.25.1)
Mako (1.0.6)
MarkupSafe (0.23)
ndg-httpsclient (0.4.0)
netaddr (0.7.19)
olefile (0.44)
pip (9.0.1)
ply (3.7)
portend (1.8)
pyasn1 (0.1.9)
pycparser (2.14)
pyOpenSSL (0.15.1)
PyPDF2 (1.26.0)
pytz (2016.10)
requests (2.9.1)
setuptools (20.7.0)
simplejson (3.8.1)
six (1.10.0)
slowaes (0.1a1)
ssdeep (3.1)
tempora (1.6.1)
typing (3.5.3.0)
unity-lens-photos (1.0)
urllib3 (1.13.1)
wheel (0.29.0)
whois (0.7)
XlsxWriter (0.9.6)
user01@recon02:~/spiderfoot$

```

**Comment by roxy9990** [comment]
I fixed the issue. Here are the commands I executed if anyone else encounters the same error:

```
sudo -H pip uninstall M2Crypto
sudo apt-get install python-m2crypto
```

**Comment by smicallef** [comment]
Thanks. I'll add this to the Dockerfile as well as updating the documentation to reflect this.

**Comment by smicallef** [comment]
Should now be addressed in 2.9.0.
